src/parts/AccountModal/index.tsx

```diff
@@ -33,6 +33,8 @@ import { shortAddress } from 'common/utils/utils';
 import { StoreType } from 'common/types/util.types';
 import { changeAddressAction } from 'common/actions/general.actions';
 import { ReactComponent as PolkadotExtensionLogoIcon } from 'assets/img/polkadot-extension-logo.svg';
+// FIXME: name clash for constants so had to use relative path
+import * as constants from '../../constants';
```

Comment:
@nud3l @tomjeatt 
Another general comment: `import *` does not look good to me because it imports everything regardless of whether all of the elements are used or not. This could impact tree-shaking and finally bundle-size. I'd like us to pick up elements we need to use.

Reply:
Yeah, I agree. I'd like to change the way we handle config items since there are different places where we have the config items.